gradio/components/dialogue.py

```diff
@@ -63,10 +65,12 @@ def __init__(
         max_lines: int | None = None,
         show_submit_button: bool = True,
         show_copy_button: bool = True,
+        ui_mode: Literal["dialogue-only", "text-only", "both"] = "both",
```

Comment:
Suggestion: shorten the parameter names ("dialogue" implies "dialogue-only" imo):

```suggestion
        ui_mode: Literal["dialogue", "text", "both"] = "both",
```